lib/index.js

```diff
@@ -30,10 +31,17 @@ Outbound.prototype.initialize = function() {
   window.outbound.methods = [
     'identify',
     'track',
+    'alias',
     'registerApnsToken',
     'registerGcmToken',
     'disableApnsToken',
     'disableGcmToken',
+    'disableAllGcmTokens',
```

Comment:
@f2prateek thanks for your feedback! is there an easy way to test against this? we tested this locally, and was able to pass all the tests.